dspy/clients/lm.py

```diff
@@ -305,17 +238,18 @@ def litellm_completion(request: Dict[str, Any], num_retries: int, cache={"no-cac
         **request,
     )
 
-
-@request_cache(maxsize=None)
 def cached_litellm_text_completion(request: Dict[str, Any], num_retries: int):
-    return litellm_text_completion(
-        request,
-        num_retries=num_retries,
-        cache={"no-cache": False, "no-store": False},
-    )
+    @dspy_cache_decorator()
+    def cached_litellm_text_completion_inner(request: Dict[str, Any], num_retries: int):
+        return litellm_text_completion(
+            request,
+            cache={"no-cache": False, "no-store": False},
```

Comment:
This passes cache to the function below. But for some reason we removed the cache kwarg below? Why?

Reply:
The default (both kwargs true) should be added back to `litellm_text_completion` function (I forgot to fix this)